TestProjects/Playground/Playground.Forms.Droid/MainActivity.cs

```diff
@@ -20,6 +22,11 @@ protected override void OnCreate(Bundle bundle)
             TabLayoutResource = Resource.Layout.Tabbar;
             ToolbarResource = Resource.Layout.Toolbar;
             base.OnCreate(bundle);
+
+            // No Splash Screen: Uncomment these lines if removing splash screen
+            // var startup = Mvx.Resolve<IMvxAppStart>();
+            // startup.Start();
+            // InitializeForms(bundle);
```

Comment:
So now you need to call this yourself in code? Why didn't the previous code work?

Reply:
It just didn't ;-)
I'm not sure why Android was different from other platforms where calling this is required. This code is consistent with iOS and UWP now.